sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -1626,6 +1625,7 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
 
   test("SPARK-33084: Add jar support Ivy URI in SQL") {
     val testData = TestHive.getHiveFile("data/files/sample.json").toURI
+    val hiveVersion = "2.3.9" // TODO remove after Hive 2.3.10 jar available in maven central
```

Comment:
Ditto.  Do you know why `DEFAULT_ARTIFACT_REPOSITORY` change isn't enough here?

Reply:
I remember it did not work on my first try, let me try again

Reply:
I searched `DEFAULT_ARTIFACT_REPOSITORY`, it looks like affects multiple places, I addressed the `ADD JAR` in another approach https://github.com/apache/spark/pull/45372/commits/ec9848f108ce529e5e4f2d55bc66169474a0eb95